local/snapshot.go

```diff
@@ -153,7 +158,7 @@ func (ln *localNetwork) persistNetwork() error {
 	// save network conf
 	networkConfig := network.Config{
 		NetworkID:          ln.networkID,
-		Genesis:            string(ln.genesis),
+		Genesis:            string(ln.genesisData),
```

Comment:
we need to persist also beacon data and upgrade file

Reply:
done